contrib/pg_tde/src/catalog/tde_principal_key.c

```diff
@@ -447,15 +445,99 @@ clear_principal_key_cache(Oid databaseId)
  * SQL interface to set principal key
  */
 
+Datum
+pg_tde_create_key_using_database_key_provider(PG_FUNCTION_ARGS)
```

Comment:
I think I would prefer to call it `generate` to differentiate it from commands which creates objects inside `pg_tde`.

Reply:
I also prefer `create`, as we are creating keys on the providers, generate wouldn't be that explicit about this.

Reply:
I disagree but seems like I am outvoted. :) I think generate makes the usage more clear.